openquakeplatform/test/versions_test.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+import unittest
+from openquakeplatform.test import pla
+
+
+class AppVersionsTest(unittest.TestCase):
+
+    def page_app_versions_test(self):
+
+        versions_url = '/versions'
+
+        # redirect page of versions application
+        pla.get(versions_url)
+
+        pla.xpath_finduniq(
+            "//h2[@class='page-title'"
+            " and normalize-space(text())='Application Versions']")
```

Comment:
Please check that at least one of the plugin application is recognized by the page

Reply:
Fixed here: https://github.com/gem/oq-platform2/pull/20/commits/ea629723cc940b2ca3bfb3be890adee142d7db3b